### DISCUSSION TITLE
The blocks do not add the level of the island

### DISCUSSION THREAD

**Discussion by xFlike** [BG-Software-LLC/SuperiorSkyblock2#1422] [Pending Feature Requests] [answered]
I assigned a value to the given block. I put it on my island he doesn't raise the level of the island why? Version 1.19
![123Bez tytułu](https://user-images.githubusercontent.com/116177569/196702645-cc6b1214-69b3-4806-8d59-4579555b31ff.png)

**Comment by xFlike** [comment]
Help me please

**Comment by OmerBenGera** [comment, marked as answer]
Open an issue regarding this instead of a feature request.
Before doing so tho, ask for help in the discord server.
